#39 CourseInfo component

JorgenHagglund opened the pull request:
#### PR Summary
This pull request introduces new UI components and updates existing functionality to improve the user experience in the application. Key changes include:
- `CourseInfo.razor`: Added UI components for displaying course information, including a student list.
- `FaderBox.razor`: Created a new component for displaying text with a "read more" button.
- `UsersController.cs`: Modified authorization for `GetUserById` to allow access for any authenticated user.

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Created a fading test-box
Created CourseInfo component
Created StudentList component

JorgenHagglund committed:
Refactored the CourseController/Service/Repository to include "IncludeUsers" parameter for getCoursesForUser()

JorgenHagglund committed:
Refactored the CourseController/Service/Repository to include "IncludeUsers" parameter for getCoursesForUser()

JorgenHagglund committed:
Refactored the CourseController/Service/Repository to include "IncludeUsers" parameter for getCoursesForUser()

JorgenHagglund committed:
Refactored the CourseController/Service/Repository to include "IncludeUsers" parameter for getCoursesForUser()

JorgenHagglund committed:
Refactored the CourseController/Service/Repository to include "IncludeUsers" parameter for getCoursesForUser()

JorgenHagglund committed:
Refactored the CourseController/Service/Repository to include "IncludeUsers" parameter for getCoursesForUser()

JorgenHagglund committed:
Refactored the CourseController/Service/Repository to include "IncludeUsers" parameter for getCoursesForUser()

JorgenHagglund committed:
Refactored the CourseController/Service/Repository to include "IncludeUsers" parameter for getCoursesForUser()

JorgenHagglund committed:
Merge branch 'Component_CourseInfo' of https://github.com/PhilippeLeopoldie/LexiconLMS into Component_CourseInfo

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Corrected parameter handling in ProxyController
Adjusted css's and components

JorgenHagglund committed:
Minor design changes

JorgenHagglund committed:
Minor design changes

JorgenHagglund committed:
Minor design changes

JorgenHagglund committed:
Minor design changes

JorgenHagglund committed:
Minor design changes

JorgenHagglund committed:
Minor design changes

JorgenHagglund committed:
Minor design changes

JorgenHagglund committed:
Minor design changes

JorgenHagglund committed:
Stage before pull

JorgenHagglund committed:
Copied ProxyController.cs from development

JorgenHagglund committed:
Added Spinner to CourseInfo component.
Added link in StudentCard component.
Added image for StudentCard.
Relaxed restrictions in UserController so that all authorized can get user details.

JorgenHagglund committed:
Added Spinner to CourseInfo component.
Added link in StudentCard component.
Added image for StudentCard.
Relaxed restrictions in UserController so that all authorized can get user details.

JorgenHagglund committed:
Added Spinner to CourseInfo component.
Added link in StudentCard component.
Added image for StudentCard.
Relaxed restrictions in UserController so that all authorized can get user details.

JorgenHagglund committed:
Added Spinner to CourseInfo component.
Added link in StudentCard component.
Added image for StudentCard.
Relaxed restrictions in UserController so that all authorized can get user details.

JorgenHagglund committed:
Added Spinner to CourseInfo component.
Added link in StudentCard component.
Added image for StudentCard.
Relaxed restrictions in UserController so that all authorized can get user details.

JorgenHagglund committed:
Added Spinner to CourseInfo component.
Added link in StudentCard component.
Added image for StudentCard.
Relaxed restrictions in UserController so that all authorized can get user details.

JorgenHagglund committed:
Added Spinner to CourseInfo component.
Added link in StudentCard component.
Added image for StudentCard.
Relaxed restrictions in UserController so that all authorized can get user details.

JorgenHagglund committed:
Merge branch 'Component_CourseInfo' of https://github.com/PhilippeLeopoldie/LexiconLMS into Component_CourseInfo

JorgenHagglund committed:
Trying to get FaderBox to fade into background

JorgenHagglund committed:
Trying to get FaderBox to fade into background

JorgenHagglund committed:
Trying to get FaderBox to fade into background

JorgenHagglund committed:
Trying to get FaderBox to fade into background

JorgenHagglund committed:
Trying to get FaderBox to fade into background

JorgenHagglund committed:
Trying to get FaderBox to fade into background

JorgenHagglund committed:
Tweaked Home.cs to show user's name.
Played around with the design of Student Lists/Cards for Courses component

JorgenHagglund committed:
Tweaked Home.cs to show user's name.
Played around with the design of Student Lists/Cards for Courses component

JorgenHagglund committed:
Tweaked Home.cs to show user's name.
Played around with the design of Student Lists/Cards for Courses component

JorgenHagglund committed:
Tweaked Home.cs to show user's name.
Played around with the design of Student Lists/Cards for Courses component

JorgenHagglund committed:
Tweaked Home.cs to show user's name.
Played around with the design of Student Lists/Cards for Courses component

JorgenHagglund committed:
Tweaked Home.cs to show user's name.
Played around with the design of Student Lists/Cards for Courses component

JorgenHagglund committed:
Changed a prompt to swedish.
Tweaked another prompt.

JorgenHagglund committed:
Changed a prompt to swedish.
Tweaked another prompt.

JorgenHagglund committed:
Changed a prompt to swedish.
Tweaked another prompt.

JorgenHagglund committed:
Changed a prompt to swedish.
Tweaked another prompt.

JorgenHagglund committed:
Changed a prompt to swedish.
Tweaked another prompt.

JorgenHagglund committed:
Changed a prompt to swedish.
Tweaked another prompt.

JorgenHagglund committed:
Merged with development branch

JorgenHagglund committed:
Added jQuery client-side library (sorry!)
Added code for read-more (in FaderBox)

JorgenHagglund committed:
Added jQuery client-side library (sorry!)
Added code for read-more (in FaderBox)

JorgenHagglund committed:
Added jQuery client-side library (sorry!)
Added code for read-more (in FaderBox)

JorgenHagglund committed:
Added jQuery client-side library (sorry!)
Added code for read-more (in FaderBox)

JorgenHagglund committed:
Fixed conflicts

JorgenHagglund committed:
Changed link to where to go when a user card is clicked.

JorgenHagglund committed:
Changed link to where to go when a user card is clicked.

JorgenHagglund committed:
Changed link to where to go when a user card is clicked.

JorgenHagglund committed:
Changed link to where to go when a user card is clicked.

JorgenHagglund committed:
Rebased

PhilippeLeopoldie committed:
Add module creation/editing form in ModuleForm.razor

This update introduces a new form for creating or editing modules. It features an `EditForm` component bound to a `FormModel`, with fields for the module's name, description, start date and time, and end date and time, each accompanied by validation messages. A validation summary and a submit button are included, along with parameters for the `FormModel` and an event callback for form submission.

PhilippeLeopoldie committed:
Replace EditForm with ModuleForm in CreateModule.razor

Updated the form implementation by replacing the `<EditForm>` component with a new `<ModuleForm>` component, maintaining the same `FormModel` and `OnValidSubmit` parameters. Retained error message display for better user feedback, while keeping the access control within the `<AuthorizeView>` for "Teacher" role users.

PhilippeLeopoldie committed:
Update module display to include start and end dates

The `CourseRow` component in `ModulesComponent.razor` now shows the module's start and end dates alongside its name. This enhances the information presented to users, while the activity count remains unchanged.

PhilippeLeopoldie committed:
Enforce required Description in DTOs

Updated the `Description` property in `ModuleDto` and `ModuleForManipulationDto` from nullable to required. Added validation to ensure that a description must be provided, enhancing data integrity for these data transfer objects.

PhilippeLeopoldie committed:
Add generic Result<T> class for operation outcomes

Introduced a new generic class `Result<T>` in the
`LMS.Blazor.Client.Pages.Modules` namespace. This class
provides properties for operation success, result data,
and error messages. It includes a private constructor
and two static methods: `Ok(T data)` for successful
results and `Fail(string errorMessage)` for failures.

JorgenHagglund committed:
Fised 500-error when POST'ing course.

JorgenHagglund committed:
Merge branch 'development' of https://github.com/PhilippeLeopoldie/LexiconLMS into development

ahlgren1234 committed:
Skapa kurs.

ChornaIryna commented on LMS.Blazor/wwwroot/app.css:
Det verkar som att du har en gammal version av app.css här.

PhilippeLeopoldie committed:
Refactor ModuleComponent for improved UI and functionality

Renamed `courseId` to `CourseId` for consistency.
Introduced `ModuleMenu` method for dropdown actions.
Restructured UI components for better layout control.
Updated API calls to use the new `CourseId` property.

PhilippeLeopoldie committed:
Add ModuleStateService for module state management

Introduces `ModuleStateService` to the dependency injection
container in `Program.cs`, allowing for scoped management of
module state within the Blazor client. A new file,
`ModuleStateService.cs`, defines the service and its
`Module` property of type `ModuleDto?`. Existing service
registrations for API and authentication have been modified
to reflect this change.

PhilippeLeopoldie committed:
Add CourseId parameter to ModuleForm and refactor submit

Updated the `ModuleForm` in `CreateModule.razor` to include a new `CourseId` parameter for better context. Refactored the `HandleValidSubmit` method to improve clarity by moving navigation logic to a new `NavigateBack` method. Error handling remains unchanged to ensure proper exception management.

PhilippeLeopoldie committed:
Replace ModulesComponent with ModuleComponent

Updated the Razor component to replace the `ModulesComponent` with a `ModuleComponent`, indicating a shift to focus on a single module representation. The surrounding context and header actions remain unchanged.

PhilippeLeopoldie committed:
Enhance ModuleForm with improved structure and navigation

Updated `ModuleForm.razor` to use an `EditForm` for better validation handling. Restructured labels and input fields, adding `SlutDatum` (End Date) and `Beskrivning` (Description). Improved layout with Bootstrap classes and added a "Cancel" button for navigation. Introduced a new `CourseId` parameter and a `NavigateBack` method for easier navigation to course modules.

PhilippeLeopoldie committed:
Add customizable menu content to CourseRow component

Introduced a new `MenuContent` parameter of type `RenderFragment?` to the `CourseRow` component for customizable menu display. Conditional rendering is implemented to show `MenuContent` in a `div` if not null, or a button to trigger the menu if `MenuContent` is null but `HasMenu` is true.

PhilippeLeopoldie committed:
Add EditModule page with authorization and form handling

Updated `EditModule.razor` to create a new page for editing modules within a course. Added authorization for "Teacher" role, a form for editing module details, and error handling for submissions. Initialized form model with existing module data and implemented API update requests. Removed unused method for fetching modules by course ID.

ahlgren1234 committed:
Merge pull request #111 from PhilippeLeopoldie/#34-form-to-create-course

Skapa kurs.

PhilippeLeopoldie committed:
EditModule: Update header with authorization check

Replaced the static header with an `<AuthorizeView>` component to ensure only users with the "Teacher" role can see the module name.

PhilippeLeopoldie committed:
Merge branch 'development' into #33-1-implement-EditModule

PhilippeLeopoldie committed:
Remove ActionName attribute from GetCourseByIdAsync

The `ActionName` attribute for the `GetCourseByIdAsync` method was removed from the `CoursesController` class. This change may affect how the action is referenced in routes or when generating links, as the explicit action name is no longer defined.

ahlgren1234 reviewed (approved):
Ser bra ut.

JorgenHagglund committed:
Solved conflicts

JorgenHagglund committed:
Fixed a "Duplicate ActionName" error that arose from rebasing.

JorgenHagglund committed:
Copied app.css from development branch

JorgenHagglund committed:
Copied app.css from development branch

PhilippeLeopoldie committed:
Merge pull request #113 from PhilippeLeopoldie/#33-1-implement-EditModule

#33 1 implement edit module

ChornaIryna reviewed (commented):
Koden ser bra ut, så jag förstår inte varför jag får den där vyn.: 
<img width="1878" height="925" alt="image" src="https://github.com/user-attachments/assets/9d802227-f0b4-4f2d-994c-e514acc1de54" />

PhilippeLeopoldie commented:
> Koden ser bra ut, så jag förstår inte varför jag får den där vyn.: <img alt="image" width="1878" height="925" src="https://private-user-images.githubusercontent.com/148462332/486039980-9d802227-f0b4-4f2d-994c-e514acc1de54.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTcwNjUyMDksIm5iZiI6MTc1NzA2NDkwOSwicGF0aCI6Ii8xNDg0NjIzMzIvNDg2MDM5OTgwLTlkODAyMjI3LWYwYjQtNGYyZC05OTRjLWU1MTRhY2MxZGU1NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwOTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDkwNVQwOTM1MDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yNTNhMjliMDUzZjQ5NDJjMGFlYmVkY2JlMGZhOTg1ZDA3YTg2MDI1NGZkNTZkOWVjZmU4ZDUzZTI1MTg2NTA2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.sxiRH-2Zez1x9rM6G9XCsHI3B5yHm4pICnS3xGnLLmE">

Jag får samma problem men bara med ett elevkonto, med kurskomponenten, när skärmen blir smalare än 1110 px.

PhilippeLeopoldie commented on LMS.Blazor.Client/Shared/Components/StudentCard.razor:
Det kan se bättre ut om alla kort har en fast bredd, och med fullständigt namn istället för e-postadressen. Vad tror du?

JorgenHagglund replied:
Fullständigt namn står till höger om bilden. 
Egentligen tänkte jag att e-post adressen också skulle vara till höger, men det får jag inte till.

PhilippeLeopoldie replied:
Så klart, jag har inte uppdaterat min databas med namn, därför kunde jag inte se det

JorgenHagglund commented:
> Koden ser bra ut, så jag förstår inte varför jag får den där vyn

Jag kan inte replikera det... Om det inte är överlappningen Du menar? Den förstår jag inte...

JorgenHagglund committed:
Corrected a small design-flaw

JorgenHagglund committed:
Corrected a small design-flaw

JorgenHagglund committed:
Fixed conflicts
